### ISSUE TITLE
Prevent Escape key from exiting the program

### ISSUE THREAD

**Issue by SkepticalSnail** [OV2/RapidCRC-Unicode#184]
This is unexpected behavior, and potentially wastes a lot of user's time if hit accidentally after hashing a big file or lots of files (ask me how I know :rofl:). No other program does this. The convention is for Escape to close dialog windows, but not the program itself.

**Comment by vlakoff** [comment]
I agree with this. The Escape key should always act as a kind of “panic button” that users can press without risking the loss of work or interrupting long operations.

As a side note, [WinMerge](https://winmerge.org/) also allows closing child windows — and even the whole program — with Escape, but that behavior is fully configurable.

At first, I thought the most appropriate approach would be to remove the Escape‑to‑exit behavior. But on second thought, some users might actually appreciate being able to close RapidCRC quickly that way. So it would probably make more sense to make this behavior configurable instead.
